tests/testthat/test-metadata.R

```diff
@@ -651,3 +749,51 @@ test_that("xportr_*: Domain is kept in between calls", {
   expect_equal(attr(df5, "_xportr.df_arg_"), "adsl")
 })
 # end
+
+test_that("`xportr_metadata()` results match traditional results", {
+  if (require(magrittr, quietly = TRUE)) {
+    test_dir <- tempdir()
+
+    trad_path <- file.path(test_dir, "adsltrad.xpt")
+    metadata_path <- file.path(test_dir, "adslmeta.xpt")
+
+    dataset_spec_low <- setNames(dataset_spec, tolower(names(dataset_spec)))
+    names(dataset_spec_low)[[2]] <- "label"
+
+    var_spec_low <- setNames(var_spec, tolower(names(var_spec)))
+    names(var_spec_low)[[5]] <- "type"
+
```

Comment:
Same suggestion as above to remove messages from tests

This can only be safely done if we're not comparing the messages themselves

```suggestion

  withr::local_message_sink(withr::local_tempfile())
```